themes/classic/templates/_partials/microdata/product-jsonld.tpl

```diff
@@ -37,10 +37,10 @@
       "@type": "Thing",
       "name": "{if $product_manufacturer->name}{$product_manufacturer->name|escape:'html':'UTF-8'}{else}{$shop.name}{/if}"
     },{/if}
-    {if isset($nbComments) && $nbComments && $ratings.avg}"aggregateRating": {
+    {if isset($product.productComments) && !empty($product.productComments.numComments) && !empty($product.productComments.nbComments)}"aggregateRating": {
```

Comment:
```suggestion
    {if !empty($product.productComments.numComments) && !empty($product.productComments.nbComments)}"aggregateRating": {
```

Reply:
When using the empty method, you're also testing if it exists. 
![image](https://user-images.githubusercontent.com/1462701/129595832-e9b80518-0635-46ae-b75d-867d87177232.png)
It's redundant and cost memory usage for nothing

Reply:
Fixed